templates/aframe-model-template/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "aframe-model-template",
+  "description": "A WebVR scene. Made with A-Frame, a WebVR framework.",
+  "author": "A-Frame authors",
+  "version": "0.1.1",
+  "license": "CC0-1.0",
+  "repository": "aframevr-userland/aframe-default-template",
```

Comment:
Oh gotcha.  OK, I wasn't sure where to put these and didn't catch aframevr-userland.   Hence the PR's.   

I'll start moving into that repo.